src/modules/musig/Makefile.am.include

```diff
@@ -1,16 +1,8 @@
 include_HEADERS += include/secp256k1_musig.h
 noinst_HEADERS += src/modules/musig/main_impl.h
-noinst_HEADERS += src/modules/musig/tests_impl.h
-
-noinst_PROGRAMS += example_musig
-example_musig_SOURCES = src/modules/musig/example.c
-example_musig_CPPFLAGS = -DSECP256K1_BUILD -I$(top_srcdir)/include $(SECP_INCLUDES)
-if !ENABLE_COVERAGE
-example_musig_CPPFLAGS += -DVERIFY
-endif
-example_musig_LDADD = libsecp256k1.la $(SECP_LIBS)
-example_musig_LDFLAGS = -static
-
-if USE_TESTS
-TESTS += example_musig
-endif
+noinst_HEADERS += src/modules/musig/keyagg.h
+noinst_HEADERS += src/modules/musig/keyagg_impl.h
+noinst_HEADERS += src/modules/musig/session.h
+noinst_HEADERS += src/modules/musig/session_impl.h
+noinst_HEADERS += src/modules/musig/adaptor_impl.h
+noinst_HEADERS += src/modules/musig/tests_impl.h
```

Comment:
micronit: GitHub says "no newline at end of file" (which is probably not an issue at all but should be avoided)

Reply:
should be fixed